askama_shared/src/generator.rs

```diff
@@ -1058,11 +1058,12 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
                             expr_buf.buf, self.input.escaper
                         ),
                     };
+                    let is_cacheable = !matches!(s, Expr::Call(..));
```

Comment:
I'm not sure if this doesn't need to be a "deep" check. E.g. `foo().0` should still be uncachable, right?

Reply:
Oh yeah, you're right. Guess my quick fix, was too quick. I'll fix it later 😅

Reply:
Should I review before you fix it or do you want to fix it before I review? Not sure what's more appropriate here.